fe/fe-common/src/main/java/org/apache/doris/catalog/PrimitiveType.java

```diff
@@ -620,6 +621,9 @@ public static ImmutableSetMultimap<PrimitiveType, PrimitiveType> getImplicitCast
         builder.put(TIMEV2, DATEV2);
         builder.put(TIMEV2, DATETIMEV2);
 
+        // VARBINARY
+        builder.put(VARBINARY, VARBINARY);
```

Comment:
this matrix is deperated, u should update similar matrix in nereids, @Jibing-Li PTAL